public/index.html

```diff
@@ -1,28 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
I don't see any changes in the file.  only some indentation mixups. Can be removed from the pull request?!

Reply:
I also unstaged the changes here. Must have been some accidental indentation when I opened the file some time! 😅